serverscommands/keypaircommands/create.go

```diff
@@ -29,13 +30,15 @@ func flagsCreate() []cli.Flag {
 	return []cli.Flag{
 		cli.StringFlag{
 			Name: "publicKey",
-			Usage: `[optional] The public ssh key to associate with the user's account.
-	It may be the actual key or the file containing the key. If empty,
-	the key will be created for you and returned in the output.`,
+			Usage: strings.Join([]string{"[optional] The public ssh key to associate with the user's account.",
+				"It may be the actual key or the file containing the key. If empty,",
+				"the key will be created for you and returned in the output."}, "\n\t"),
```

Comment:
Looks good.

```
12:02:06 (limit-fields=) kyle6475@puter ~/go/src/github.com/jrperritt/rack$ go build -o $GOPATH/bin/rack && rack servers keypair create -h
NAME:
   create - rack servers keypair create <keypairName> [flags]

USAGE:
   command create [command options] [arguments...]

DESCRIPTION:
   Creates a keypair

OPTIONS:
   --publicKey 	[optional] The public ssh key to associate with the user's account.
		It may be the actual key or the file containing the key. If empty,
		the key will be created for you and returned in the output.
   --fields 	Only return these comma-separated case-insensitive fields for each item in the list.
		Choices: name, fingerprint, publickey, privatekey
```